securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
@dachary I'm not suggesting you re-integrate them; I think separate PR's is definitely the way to go. However, I think that you should remove irrelevant changes from this PR, such as these new `args` parameters. Does that make sense?